modules/light-clients/07-tendermint/types/client_state.go

```diff
@@ -19,6 +19,11 @@ import (
 	"github.com/cosmos/ibc-go/modules/core/exported"
 )
 
+// Expired is a potential status of a client. A client is expired
+// if its latest consensus state timestamp added to its trusting period
+// is before or equal to the current time.
+const Expired = "Expired"
```

Comment:
nit, this should be defined in exported too?

Reply:
My reasoning is that this is a tendermint specific concept. Is expiry common amongst light clients? cc @AdityaSripal @cwgoes

Reply:
Expiry will be common to light clients other than Tendermint, yes. Anything for proof-of-stake should have this.